bin/crew

```diff
@@ -51,6 +51,13 @@ require_gem 'pry-byebug' if CREW_DEBUG
 # being started with a --debug flag.
 # The following line can be used anywhere as a breakpoint.
 binding.pry if CREW_DEBUG
+
+begin
+  require_relative '../lib/crew_mvdir'
+rescue LoadError
+  warn 'crew-mvdir is not installed. Please install it with \'crew install crew_mvdir\' for improved installation performance'.yellow
```

Comment:
`for improved package installation performance.`?  End with period.

Reply:
Change made